setup.py

```diff
@@ -22,7 +22,7 @@ def read(*paths):
 INSTALL_REQUIRES = [
     # General
     'six>=1.9,<2.0',
-    'click>=6.0,<7.0',
+    'click>=7.0,<8.0',
```

Comment:
@akariv 
Is there a reason to drop `click@6`?

Reply:
Not really - fixing